soc/espressif/Kconfig

```diff
@@ -56,4 +56,37 @@ config RTC_CLK_CAL_CYCLES
 	    If the crystal could not start, it will be switched to internal RC.
 endmenu
 
+menu "Sleep Config"
```

Comment:
Can we remove this fix from this PR, please? There is a draft in https://github.com/zephyrproject-rtos/hal_espressif/pull/311